index.js

```diff
@@ -4,7 +4,8 @@ const port = 5000;                  //Save the port number where your server wil
 
 //Idiomatic expression in express to route and respond to a client request
 app.get('/', (req, res) => {        //get requests to the root ("/") will route here
-    res.sendFile('hello.html', {root: __dirname});      //server responds by sending the index.html file to the client's browser                                                 //the .sendFile method needs the absolute path to the file, see: https://expressjs.com/en/4x/api.html#res.sendFile 
+    res.sendFile('hello.html', {root: __dirname});
+    console.log(`Website accessed`);       //server responds by sending the index.html file to the client's browser                                                 //the .sendFile method needs the absolute path to the file, see: https://expressjs.com/en/4x/api.html#res.sendFile 
```

Comment:
Codacy has a fix for the issue: [Extra semicolon.](https://app.codacy.com/gh/PetteriV99/Group-Assignment-6-CI-CD/pullRequest?prid=9293795)
```suggestion
    console.log(`Website accessed`)       //server responds by sending the index.html file to the client's browser                                                 //the .sendFile method needs the absolute path to the file, see: https://expressjs.com/en/4x/api.html#res.sendFile 
```